store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9539,4 +9539,9 @@ TODO: delete them permanently from here
 <attr id="3018" name="zimbraPreviousEphemeralBackendURL" type="string" cardinality="single" optionalIn="globalConfig" immutable="1" since="8.8.1">
   <desc>URL of the previous ephemeral storage backend</desc>
 </attr>
+
+<attr id="3019" name="zimbraAttributeMigrationInfo" type="string" cardinality="single" optionalIn="globalConfig" immutable="1" since="8.8.1">
```

Comment:
This can't really be an enum because it's an encoded string containing the status, destination URL, and timestamp.
Regarding immutability: If there are SOAP tests that fail due to the presence of immutable attributes, that seems like an underlying issue with the tests themselves. AFAIK, attributes should be made immutable when they store some internal server state, which is the case here.
That said, I'm happy to remove the immutable constraint for the time being if it's likely to cause problems.